vesta/chars.py

```diff
@@ -161,18 +161,18 @@ def encode_text(
     max_cols = COLS - margin * 2
     rows: List[List[int]] = []
 
-    def find_break(line: List[int]) -> int:
+    def find_break(line: list[int]) -> tuple[int, bool]:
         end = min(len(line), max_cols)
         for pos in range(end, 0, -1):
             if line[pos] in breaks:
-                return pos
-        return end
+                return pos, True
+        return end, False
 
     for line in map(encode, s.splitlines()):
         while len(line) > max_cols:
-            pos = find_break(line)
+            pos, found = find_break(line)
```

Comment:
Do you think it would be clearer if `find_break()` returned a `Tuple[int, int]` representing the "break" and "resume" positions? That would make the logic in the outer loop a bit nicer:

```python
pos, resume = find_break(line)
rows.append(_format_row(line[:pos], align, margin, fill))
line = line[resume:]
```

Reply:
I think it's sixes, but it does indeed make it cleaner in the outer loop. That's the exact edit I just made to the code ^_^